frontend/mulighetsrommet-veileder-flate/src/apps/modia/views/Landingsside.tsx

```diff
@@ -5,18 +5,26 @@ import styles from "./Landingsside.module.scss";
 import { HistorikkKort } from "../historikk/HistorikkKort";
 import { useHistorikkFraKomet } from "../../../core/api/queries/useHistorikkFraKomet";
 import { UtkastKort } from "../historikk/UtkastKort";
+import { ErrorBoundary } from "react-error-boundary";
+import { ErrorFallback } from "../../../utils/ErrorFallback";
 
 export function Landingsside() {
   return (
     <main className="mulighetsrommet-veileder-flate">
       <VStack gap="10" className={styles.container}>
-        <Utkast />
+        <ErrorBoundary FallbackComponent={ErrorFallback}>
+          {" "}
+          {/** TODO Bedre komponent for fallback i error boundary */}
+          <Utkast />
+        </ErrorBoundary>
```

Comment:
Appen bør ikke krasje hvis vi ikke kan hente deltakelser. Veilder må fortsatt kunne navigere til arbeidsmarkedstiltakene, men `ErrorFallback` er for generell så det vi designes noe finere feilmelding for disse casene tenker jeg.